.circleci/config.yml

```diff
@@ -99,6 +99,9 @@ jobs:
           name: Run wiremock
           command: java -jar wiremock.jar --port 9091
           background: true
+      - run:
```

Comment:
Might as well do this before installing Wiremock?

Reply:
agreed